.github/workflows/release-binaries.yml

```diff
@@ -33,3 +33,19 @@ jobs:
       rust_log: info
     # This step needs access to Docker Hub secrets to run successfully
     secrets: inherit
+
+  # The image will be named `zebrad-mining-rpcs-testnet:<semver>`
+  build-mining-testnet:
+    name: Build Release Testnet Mining Docker
+    uses: ./.github/workflows/build-docker-image.yml
+    with:
+      dockerfile_path: ./docker/mining.Dockerfile
+      dockerfile_target: runtime
+      image_name: zebrad-mining-rpcs-testnet
```

Comment:
This is experimental code, but we're publishing it alongside production images. So I'd like to make it clear that it's not for production.

It might be better to do this by adding `.experimental` to the version tag, so that we don't have to maintain image names with and without `-experimental` after it gets audited.

```
      image_name: zebrad-mining-rpcs-testnet-experimental
```

Reply:
I think adding `flavour` in [`ced6cbe`](https://github.com/ZcashFoundation/zebra/pull/6228/commits/ced6cbeb8317592d236fb890ab4de74868143aec#diff-5b40193e09025fd041de92d38c976d0b469e3ee85bef45a71d05d8b5078b1ee8R86) should fix this.

Reply:
I'm not sure if adding a tag helps, because after the stable release, the version will be `1.0.0`, which makes it look like supported production code. Instead, I'd like the version to be `1.0.0-experimental` or something similar, to show it is still pre-release quality.

Currently, it's not release-candidate quality code, so I guess we should be using `1.0.0-rc.6.experimental` or `1.0.0-experimental`. But the exact format is less important.

Here's the semantic versioning reference for pre-release code:
https://semver.org/#spec-item-9